src/partest/scala/tools/partest/nest/Runner.scala

```diff
@@ -758,128 +758,16 @@ object Properties extends scala.util.PropertiesTrait {
 
 /** Used by SBT- and ConsoleRunner for running a set of tests. */
 class SuiteRunner(
+  val config: RunnerSpec.Config,
   val testSourcePath: String, // relative path, like "files", or "pending"
   val fileManager: FileManager,
-  val updateCheck: Boolean,
-  val failed: Boolean,
-  val noexec: Boolean,
   val nestUI: NestUI,
   val javaCmdPath: String = PartestDefaults.javaCmd,
   val javacCmdPath: String = PartestDefaults.javacCmd,
   val scalacExtraArgs: Seq[String] = Seq.empty,
   val javaOpts: String = PartestDefaults.javaOpts,
   val scalacOpts: String = PartestDefaults.scalacOpts) {
 
-  import PartestDefaults.{ numThreads, waitTime }
-
-  setUncaughtHandler
-
-  // TODO: make this immutable
-  PathSettings.testSourcePath = testSourcePath
-
-  val durations = collection.concurrent.TrieMap[File, Long]()
```

Comment:
Does no one report on longest-running tests? or is that in sbt?

Reply:
I've seen some individual tests report durations (if it's over a set threshold I assume) but apparently we don't use these collected durations anywhere.